core/domain/rating_services.py

```diff
@@ -76,8 +75,6 @@ def _update_user_rating():
 
     exploration_summary = exp_services.get_exploration_summary_by_id(
         exploration_id)
-    if not exploration_summary.ratings:
```

Comment:
Why delete?

Reply:
When we create an exploration the ratings that are assigned to it is by `feconf.get_empty_ratings()`. Hence the statement below has no use since it already has ratings equal to `feconf.get_empty_ratings()`

Reply:
You can look at [this function](https://github.com/oppia/oppia/blob/develop/core/domain/exp_services.py#L1217) for clarification.

Reply:
I think the main thing I'm concerned about is that that default might have been introduced only after things had run in production for a while, so it's possible that there are explorations in production that have ratings = None.

So we could do two things. Either we tackle this in tests. Or we write a job that iterates through all the exploration summary models in production and verifies that the ratings field is not None. Which would you prefer?

Reply:
Done!